src/quesadilla.md

```diff
@@ -1,30 +1,35 @@
 # Quesadilla
 
-A quick and easy meal, able to be taken anywhere, but be warned of its high empty carb counts.
+![brisket quesadilla](pix/quesadilla.webp "Brisket and Bean Quesadilla with using Montery Jack Cheese")
```

Comment:
Correct me if I'm wrong but is this valid image syntax? ![description](file "string")

Reply:
Yes it is, the part in quotes represents what you see when you mouse over it https://raw.githubusercontent.com/LukeSmithxyz/based.cooking/master/src/baked-mostaccioli.md

Reply:
Hmm I see, wonder if this standard should be enforced once we start cleaning up...

Reply:
It's too obscure I think. I would leave it optional. The only time people use that feature is for xkcd comics...